hs_core/templates/debug/debug_resource.html

```diff
@@ -135,4 +139,66 @@ <h2>iRODs Issues</h2>
   };
   xhttp.open("GET", "irods-issues/", true);
   xhttp.send();
+
+  // Load async data for non preferred paths in the resource
+  let xhttp_path_check = new XMLHttpRequest();
```

Comment:
many of these `let` declarations could use `const`
not required but recommended where possible

Reply:
Fixed